SphereAPIDeviceCodeAuth/README.md

```diff
@@ -0,0 +1,20 @@
+# Access Azure Sphere API using AAD Device code flow
+This sample code snippet can be used to access Azure Sphere public API using AAD token acquired by AAD Device code flow. This code can be used to acquire token in Web application context.
+
+## Contents
+
+| File/folder | Description |
+|-------------|-------------|
+| `DeviceCodeModel.cshtml`       | The Razor PageModel |
+| `DeviceCodeModel.cshtml.cs` | The PageModel backend. |
+| `README.md`   | This README file. |
+| `LICENSE.txt`   | The license for the project. |
+
+## Concept
+Please refer the following document.
+- AAD Device Code Flow [Documentation](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-device-code)
+- Azure Sphere Public API - [Authentication values](https://docs.microsoft.com/rest/api/azure-sphere/#authentication-library-values)
+
+## How to use
+- Copy the razor pages directly in ASP .net application
+- Otherwise, use the frontend & backend in the existing application at the corresponding place.
```

Comment:
Please include sections from the README template:

```

## Project expectations

Describe the state of the code. For example:

* The code is not secure (for example, connection strings are included in the code).
* The code is grossly inefficient (for example, loops that run unnecessarily because it uses the wrong breakout points).
* The code does not run (for example, there are build errors).
* The code may cause a problem if used in a production scenario.
* The code has not been run through a complete security analysis (for example, there are static analysis failures).

### Expected support for the code

<!---If you will reply to issues, please suggest how users should report problems or reach out. Github issues is preferable.--->

How often will you check and reply to issues or maintain and update the code? For example:
* Will reply to issues once per quarter, once a year, etc.
* No support whatsoever, code is as is
```

e.g. consider:

```
## Project expectations
This code is an illustrative snippet only, it has not been reviewed for production-readiness.

### Expected support for the code
This code is not formally maintained, but we will make a best effort to respond to/address any issues you encounter.

### How to report an issue
If you run into an issue with this code, please open a GitHub issue against this repo.
```